### PR TITLE
Venture Directory

### PR DESCRIPTION
**Pull request by S-Makrod**
- VB directory implemented
- Lint fixes
- Git hooks implemented for dev experience improvement (i noticed we were forgetting about running lint before merges so now we have a remined every push)

### CODE REVIEW

**Comment by S-Makrod** [comment]
[[FE/BE][Feature] Venture directory page](https://trello.com/c/7IVRKXx8/7-fe-befeature-venture-directory-page)

File: README.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ Navigate to `./frontend` in the terminal. You will see a file called `package.js
 
 To start the frontend project run `npm start` in the frontend folder. If there are no issues the application will start at <http://localhost:3000/>.
 
+We have a hooks folder at the root of the project responsible for creating GitHub hooks. Currently we have a single pre-push hook which runs in order to remind developers to fix their lint errors. The hooks should be configured when running `npm install` but in case you have any issues run: `git config --local core.hooksPath hooks`.
```
Comment (S-Makrod) [inline README.md:61]: I made hooks to make future dev easier, see info/setup here

File: hooks/pre-push [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```
Comment (S-Makrod) [inline hooks/pre-push:1]: hook to run lint during push

File: src/api/organizationService.ts [hunk 3]
```diff
@@ -8,8 +8,15 @@ import Service from './service'
 import { METHODS } from './utils'
 
 const organizationService = new Service<Organization>(apiRoutes.organizationOperations.baseUrl)
+export const organizationQueryPrefix = 'organization'
+export const industryQueryPrefix = 'industry'
 
-const retrieve = async (query?: string): Promise<ServiceResponse<Array<Organization>>> => {
+const retrieve = async (organization: string | null, industry: string | null): Promise<ServiceResponse<Array<Organization>>> => {
```
Comment (S-Makrod) [inline src/api/organizationService.ts:14]: updated to take query params

File: src/components/VBCards.tsx [hunk 4]
```diff
@@ -56,10 +57,10 @@ export const AboutCard = ({ title, description, image, address }: IAboutCardProp
 
 export interface IVentureCardProps {
```
Comment (S-Makrod) [inline src/components/VBCards.tsx:58]: old card from contractors, just updated to be more useful

File: src/navigation/anonRoutes.tsx [hunk 5]
```diff
@@ -1,13 +1,14 @@
-import AboutUs from '../views/anon/ventureBuildIntro/AboutUs'
-import ContactUs from '../views/anon/ventureBuildIntro/ContactUs'
-import Home from '../views/anon/ventureBuildIntro/Home'
+import AboutUs from '../views/anon/intro/AboutUs'
```
Comment (S-Makrod) [inline src/navigation/anonRoutes.tsx:1]: the venture build part was redundant so i renamed it to be cleaner

File: src/views/anon/intro/VentureDirectory.tsx [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+            initializeSearch(dispatch)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [searchParams, dispatch])
+
+    useEffect(() => {
+        if (organizationsToDisplay.length == 0)
+            setOrganizations(organizations.data)
+    }, [organizations, organizationsToDisplay])
+
+    const updateDisplay = async (org: string | null, industry: string | null, sortBy: string | null) => {
```
Comment (S-Makrod) [inline src/views/anon/intro/VentureDirectory.tsx:81]: left search to backend but sort to frontend, not sure if this is the best decision maybe we will move sort to backend later

File: src/views/anon/intro/VentureDirectory.tsx [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+        scrollToSearch()
+    }
+
+    const scrollToSearch = () => {
+        const releventDiv = document.getElementById(searchId)
+        // behavior: "smooth" parameter for smooth movement
+        releventDiv!.scrollIntoView({behavior: 'smooth'})
+    }
+
+    const initializeSearch = async (dispatch: AppDispatch) => {
```
Comment (S-Makrod) [inline src/views/anon/intro/VentureDirectory.tsx]: not sure if needed anymore tbh after @hmxxxx added the loading property to orgs

**Review by hmxxxx — CHANGES_REQUESTED** [review]
looks like the top nav header is redirecting all tabs to /community (edit: before login).

also getting error when searching by tags I believe:
```
/backend/organizations/views.py", line 46, in filter_queryset
    tags = org.split(' ')
AttributeError: 'NoneType' object has no attribute 'split'
```

also also, is there a way to not scroll up and down whenever we make a search

File: src/views/anon/intro/VentureDirectory.tsx (hunk 6 above)
Comment (hmxxxx) [reply src/views/anon/intro/VentureDirectory.tsx:81]: For MVP, I'm good with sorting on the frontend. We're getting a list of all organizations anyway, we can think about adding more querying to the list of organizations later.

File: src/views/anon/intro/VentureDirectory.tsx (hunk 7 above)
Comment (hmxxxx) [reply src/views/anon/intro/VentureDirectory.tsx]: yup, can remove this

**Comment by S-Makrod** [comment]
@hmxxxx made a pr to fix the backend bug. 

For the top nav header do you mean the navbar? If it is then I think that's not in scope of the pr

Also I think the scroll is necessary because otherwise the users may  make a search but then get confused about where their results are since they would have to scroll down manually

**Review by hmxxxx — APPROVED** [review]

**Pull request merged**